ci/nightly/pipeline.yml

```diff
@@ -654,6 +654,8 @@ steps:
     label: "Canary Deploy in Staging Cloud"
     artifact_paths: junit_*.xml
     timeout_in_minutes: 1200
+    concurrency: 1
```

Comment:
Wondering what the default is if we don't specify this...

Reply:
I think the default is 1 , but it is not mentioned explicitly here https://buildkite.com/docs/pipelines/controlling-concurrency . So I will keep it explicit.